src/components/pages/Friends/Friends.js

```diff
@@ -1,11 +1,72 @@
 import React from 'react';
+import smashRequests from '../../../helpers/data/smashRequests';
+import authRequests from '../../../helpers/data/authRequests';
+import FriendItem from '../../FriendItem/FriendItem';
 import './Friends.scss';
 
 class Friends extends React.Component {
+  state = {
+    users: [],
```

Comment:
you don't need users in state - you are breaking up the users into potentials, confirmed, pending so no need to have the full array anywhere